src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -429,6 +433,47 @@ void primitive_inst::build_deps() {
     }
 }
 
+void primitive_inst::rebuild_deps(
+    std::unordered_map<primitive_id, std::shared_ptr<primitive_inst>> const& primitives) {
+
+    _deps.resize(_dep_ids.size());
+    for (size_t i = 0; i < _dep_ids.size(); i++) {
+        if (primitives.count(_dep_ids[i]) > 0) {
+            _deps[i] = primitives.at(_dep_ids[i]);
+        } else {
+            std::cout << _dep_ids[i] << " is not found in _primitives" << std::endl;
+        }
+    }
+}
+
+void primitive_inst::rebuild_exec_deps(
+    std::list<std::shared_ptr<primitive_inst>> const& primitives) {
+
+    _exec_deps.resize(_exec_dep_ids.size());
+    for (size_t i = 0; i < _exec_dep_ids.size(); i++) {
+        bool found = false;
+        for (auto& prim_inst : primitives) {
+            if (prim_inst->id().compare(_exec_dep_ids[i]) == 0) {
+                _exec_deps[i] = prim_inst;
+                found = true;
+                break;
+            }
+        }
+        if (found == false) {
+            std::cout << "not found in _exec_order" << std::endl;
```

Comment:
here too.

Reply:
It's also replaced with `OPENVINO_ASSERT`. Thank you.